experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes/Project.toml

```diff
@@ -0,0 +1,18 @@
+[deps]
```

Comment:
Since this shares most dependencies with the other ClimaCore examples, I think we can just add `DocStringExtensions`, `KernelAbstactions`, and `SurfaceFluxes` to the `examples/ClimaCore/Project.toml`. I guess I am also okay with each test case folder having its own `Project.toml`, but we should be consistent here.

Reply:
Changing my mind here; I think it will be less convoluted to have separate projects for each experiment. This will give us greater control over dependency versioning and branches. It also follows ClimaCore.

Reply:
We can add these as we update the other experiments in future PRs.